### PR TITLE
[wasm] Allow the main.js file to keep it's name

### PR DESCRIPTION
**Pull request by pavelsavara**
This will be useful to be able to have `.mjs` and `.cjs` main files, so that NodeJs could load it properly.

### CODE REVIEW

File: src/mono/wasm/build/WasmApp.targets [hunk 1]
```diff
@@ -295,12 +295,13 @@
   <Target Name="_GenerateRunV8Script">
     <PropertyGroup>
       <WasmRunV8ScriptPath Condition="'$(WasmRunV8ScriptPath)' == ''">$(WasmAppDir)run-v8.sh</WasmRunV8ScriptPath>
+      <WasmMainJSFileName>$([System.IO.Path]::GetFileName('$(WasmMainJSPath)'))</WasmMainJSFileName>
```
Comment (radical) [inline src/mono/wasm/build/WasmApp.targets:298]:
```suggestion
      <_WasmMainJSFileName>$([System.IO.Path]::GetFileName('$(WasmMainJSPath)'))</_WasmMainJSFileName>
```

**Comment by radical** [comment]
cc @thaystg to confirm that this doesn't break debugger tests

**Review by radical — APPROVED** [review]

**Comment by pavelsavara** [comment]
I tested debugger locally.

**Pull request merged**